core/src/main/java/greencity/controller/UserProfileController.java

```diff
@@ -35,13 +35,14 @@ public class UserProfileController {
     @ApiOperation(value = "Update user profile")
     @ApiResponses(value = {
         @ApiResponse(code = 200, message = HttpStatuses.OK, response = UserProfileDto.class),
```

Comment:
And also if you decide to use 204 code, remove pls 200 code from here, I think it isn’t good to have more than one 2xx response